.changeset/famous-rats-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+'@mysten/sui.js': minor
+---
+
+Only allow import Bech32 format private key
```

Comment:
Is this correct? I thought we'd support both.

Reply:
If this is correct it's a pretty big breaking change, a lot of things can break from this

Reply:
revised